tests/conftest.py

```diff
@@ -1,19 +1,114 @@
 import json
+import os
 from pathlib import Path
 
+import numpy as np
 import pytest
 
+from diffpy.srxplanar.loadimage import LoadImage
 
+
+# ----------------------------------------------------------------------
+# Filesystem fixture following the structured format
+# ----------------------------------------------------------------------
 @pytest.fixture
 def user_filesystem(tmp_path):
     base_dir = Path(tmp_path)
+    input_dir = base_dir / "input_dir"
     home_dir = base_dir / "home_dir"
-    home_dir.mkdir(parents=True, exist_ok=True)
-    cwd_dir = base_dir / "cwd_dir"
-    cwd_dir.mkdir(parents=True, exist_ok=True)
+    test_dir = base_dir / "test_dir"
+    for d in (input_dir, home_dir, test_dir):
+        d.mkdir(parents=True, exist_ok=True)
 
-    home_config_data = {"username": "home_username", "email": "home@email.com"}
+    home_config_data = {
+        "username": "home_username",
+        "email": "home@email.com",
+    }
     with open(home_dir / "diffpyconfig.json", "w") as f:
         json.dump(home_config_data, f)
 
-    yield tmp_path
+    yield {
+        "base": base_dir,
+        "input": input_dir,
+        "home": home_dir,
+        "test": test_dir,
+    }
+
+
+# ----------------------------------------------------------------------
+# LoadImage test setup
+# ----------------------------------------------------------------------
+@pytest.fixture(scope="module")
+def example_tif():
+    """Locate the example TIFF file relative to repo root."""
+    tif_path = Path("../docs/examples/KFe2As2-00838.tif").resolve()
+    if not tif_path.exists():
+        pytest.skip(f"Example TIFF not found at {tif_path}")
+    return tif_path
+
+
+@pytest.fixture(scope="module")
```

Comment:
i am not sure what this is doing but it doesn't seem appropriate to be building a fixture using one of hte functions that we will be testing.  Just delete this.

As a heads-up, we don't do any testing in `conftest.py` we just build fixtures.